### DISCUSSION TITLE
PCSX2 Be more precise in emulator description

### DISCUSSION THREAD

**Discussion by mosquitobird11** [RetroAchievements/RAWeb#1186] [Ideas]
Hi all! I am new to this repo so I don't know exactly why this differs from what must be deployed in production but we should change the description on the PCSX2 emulator (Which should be [here](https://github.com/RetroAchievements/RAWeb/blob/master/database/seeders/EmulatorsTableSeeder.php) but it's not, so I guess the version that's in prod isn't open source?) to specify that only the Nightly Qt build of PCSX2 currently supports achievements. 

I have seen a lot of confusion and experienced it myself trying to enable RA on the most recent stable build of PCSX2.

**Comment by mosquitobird11** [comment]
I also have a ton of Laravel experience all the way since 3.0 to 9.x if there is another repo anyone wants to add me to and I would happily help out some.

**Comment by Jamiras** [comment]
Supported emulators are currently defined in [releases.dist.php](https://github.com/RetroAchievements/RAWeb/blob/master/storage/app/releases.dist.php#L116-L125), but there is a copy on the server so we can edit it when versions change without having to do a new tag/deployment of the entire web product.

The file that you've found is for the new data structures that we just recently introduced and are not currently using. It's been migrated from a previous prototype.

As for noting that only nightly versions are supported, that's a communication error. When we announced an official release date for PS2 (roughly a month in advance), the PCSX2 team said they'd have a version available for users. We assumed it would be an actual version, not just a nightly.

There isn't really a good way to indicate that a nightly version is needed. Even if we did specify that a nightly was needed in the description, it's surprising how many people don't read instructions. There's also the requirement to use the QT build and confusion around the differences between AVX2 and SSE4, which we can't link directly to, or easily explain in a blurb.

I've updated the working from "Download from" to "Download the QT nightly from". Hopefully that minimizes the confusion. Now I just need some way to remember to remove "nightly" when the next stable is released.

**Reply by mosquitobird11** [reply]
Haha that all makes sense, kudos for getting that wording clarified so quickly! This will definitely save at least someone from making the same mistake that I did. Hopefully pcsx2 is working towards a big stable release, because I had no idea all of this new UI existed at all and I am loving it. 

Thanks for the clarification about where these defns are, and thanks for the amazing work with RA
